stubs/inertia/resources/js/Pages/API/ApiTokenManager.vue

```diff
@@ -34,7 +34,7 @@
             </template>
 
             <template #actions>
-                <jet-action-message :on="createApiTokenForm.recentlySuccessful" class="mr-3">
+                <jet-action-message :on="createApiTokenFormRecentlySuccessful" class="mr-3">
```

Comment:
I wonder, should this local variable just be called `apiTokenRecentlyCreated`? Seems overly verbose. 🤷‍♂️ 
```suggestion
                <jet-action-message :on="apiTokenRecentlyCreated" class="mr-3">
```